templates/package-javascript/lib/__package-name__-view.js.template

```diff
@@ -0,0 +1,29 @@
+'use babel';
+
+export default class __PackageName__View {
+
+  constructor(serializedState) {
+    // Create root element
+    this.element = document.createElement('div');
+    this.element.classList.add('__package-name__');
+
+    // Create message element
+    message = document.createElement('div');
+    message.textContent = 'The __PackageName__ package is Alive! It\'s ALIVE!';
+    message.classList.add('message');
+    this.element.appendChild(message);
+  }
+
+  // Returns an object that can be retrieved when package is activated
+  serialize() {}
+
+  // Tear down any state and detach
+  destroy() {
+    this.element.remove();
+  }
+
+  getElement() {
+    return this.element;
+  }
+
+};
```

Comment:
semicolon is not necessary here, see *export default HoistableDeclaration*: https://github.com/eslint/eslint/issues/2194#issuecomment-88575851